### PR TITLE
Fix undo redo display bug

### PR DESCRIPTION
**Pull request by khooroko**
In this PR:
using undo/redo for commands done on currently selected person now updates info panel and nearby list panel accordingly

selecting a nearby person now causes selection of that person in person list panel as well

### CODE REVIEW

**Review by jelneo — APPROVED** [review]
LGTM (Looks good to me)

File: src/main/java/seedu/address/logic/commands/NearbyCommand.java [hunk 1]
```diff
@@ -45,7 +46,18 @@ public CommandResult execute() throws CommandException {
...
+        switch (model.getCurrentList()) {
+        case "blacklist":
+            index = Index.fromZeroBased(model.getFilteredBlacklistedPersonList().indexOf(model.getSelectedPerson()));
+            break;
+        case "whitelist":
+            index = Index.fromZeroBased(model.getFilteredWhitelistedPersonList().indexOf(model.getSelectedPerson()));
+            break;
+        default:
+            index = Index.fromZeroBased(model.getFilteredPersonList().indexOf(model.getSelectedPerson()));
+        }
```
Comment (jaivigneshvenugopal) [inline src/main/java/seedu/address/logic/commands/NearbyCommand.java:59]: Thanks for using this method for now, I'll push it up to the ListObserver class once im done with it.

**Review by jaivigneshvenugopal — APPROVED** [review]
Nice!!

**Review by lawwman — APPROVED** [review]
Approved

**Pull request merged**